Sources/Public/ImageProvider/AnimationImageProvider.swift

```diff
@@ -17,5 +17,15 @@ import Foundation
 /// Additionally custom Image Providers can be made to load images from a URL,
 /// or to Cache images.
 public protocol AnimationImageProvider {
+    
+  var ignoreInternalImageCache: Bool { get set }
+    
```

Comment:
Can we invert this to something like `cacheImages` which defaults to `true`?

```suggestion
  /// Whether or not the resulting image of this image provider can be cached by Lottie. Defaults to true.
  /// If true, Lottie may internally cache the result of `imageForAsset`
  var cacheImages: Bool { get set }
```

Reply:
Open to other naming suggestions as well. I'd just like to describe this in some way similar to "whether or not Lottie is permitted to cache the result of this image provider"

Reply:
> Can we invert this to something like `cacheEligible` which defaults to `true`?

Of course, the reason why I set the default false is not to affect the old version, if you think it is better, I think it is OK.

Reply:
> > Can we invert this to something like `cacheEligible` which defaults to `true`?
> 
> Of course, the reason why I set the default false is not to affect the old version, if you think it is better, I think it is OK.



> Open to other naming suggestions as well. I'd just like to describe this in some way similar to "whether or not Lottie is permitted to cache the result of this image provider"

Of course, if you need to default to true, cacheEligible description is more accurate.